src/main/connect/connectionManager.ts

```diff
@@ -199,6 +230,10 @@ export class ConnectionManager implements ExtensionComponent {
         return proxyConfig;
     }
 
+    private updateConnectionIcon() {
+        vscode.commands.executeCommand('setContext', 'AreCredentialsSet', this.areCredentialsSet());
```

Comment:
All functions should be lowercase. It is not Go.
```suggestion
        vscode.commands.executeCommand('setContext', 'areCredentialsSet', this.areCredentialsSet());
```